x-pack/plugins/ml/public/settings/calendars/_calendar.scss

```diff
@@ -0,0 +1,15 @@
+.mlCalendar_management {
+  background: $euiColorLightestShade;
+  min-height: 100vh;
+}
+
+.mlCalendar_list {
```

Comment:
This should probably be moved out into a separate `_list.scss` file inside the `list` sub-directory, as you've done with the `edit` directory.